tests/IVIMmodels/unit_tests/test_ivim_fit.py

```diff
@@ -169,6 +169,8 @@ def test_ivim_fit_saved(name, bvals, data, algorithm, xfail, kwargs, tolerances,
     tolerances = tolerances_helper(tolerances, ratio, data["noise"])
     [f_fit, Dp_fit, D_fit] = fit.osipi_fit(signal, bvals)
     npt.assert_allclose(data['f'], f_fit, rtol=tolerances["rtol"]["f"], atol=tolerances["atol"]["f"])
-    npt.assert_allclose(data['D'], D_fit, rtol=tolerances["rtol"]["D"], atol=tolerances["atol"]["D"])
-    npt.assert_allclose(data['Dp'], Dp_fit, rtol=tolerances["rtol"]["Dp"], atol=tolerances["atol"]["Dp"])
+    if data['f']<0.80: # we need some signal for D to be detected
```

Comment:
Just looking at code in more detail, would this be a relative tolerance thing rather than an `if`?

Reply:
Yes, I am doing that in my next project. But either way it seems silly to quantity something that is undefined (D* is f=0 and D if f=1)

Reply:
Understood. Looks good to me.